docs/errors/usage_errors.md

```diff
@@ -1238,3 +1238,105 @@ except PydanticUserError as exc_info:
 
 [related specification section]: https://typing.readthedocs.io/en/latest/spec/callables.html#unpack-for-keyword-arguments
 [PEP 692]: https://peps.python.org/pep-0692/
+
+
+
+## Unsupported type for `validate_call` {#validate-call-type}
+
+`validate_call` has some limitations on the types it can validate. This error is raised when you try to use it with an unsupported type. Currently the supported types are:
+`LambdaType, FunctionType, MethodType, functools.partial`. For the case of `functools.partial`, the function being partially applied must be one of the supported types.
```

Comment:
```suggestion
`validate_call` has some limitations on the callables it can validate. This error is raised when you try to use it with an unsupported callable. Currently the supported callables are functions (including lambdas), methods and instances of [`partial`][functools.partial]. In the case of [`partial`][functools.partial], the function being partially  applied must be one of the supported callables.
```